composeApp/src/commonMain/kotlin/ui/components/dialogs/ScaleFontSizeDialog.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2024 The ZZZ Archive Open Source Project by mrfatworm
+ * License: MIT License
+ */
+
+package ui.components.dialogs
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import org.jetbrains.compose.resources.painterResource
+import org.jetbrains.compose.resources.stringResource
+import ui.components.ZzzSlider
+import ui.components.buttons.ZzzOutlineButton
+import ui.components.buttons.ZzzPrimaryButton
+import ui.components.items.RarityItem
+import ui.theme.AppTheme
+import zzzarchive.composeapp.generated.resources.Res
+import zzzarchive.composeapp.generated.resources.apply
+import zzzarchive.composeapp.generated.resources.default_value
+import zzzarchive.composeapp.generated.resources.font_scale
+import zzzarchive.composeapp.generated.resources.img_summer_solstice_2023_artwork
+import zzzarchive.composeapp.generated.resources.multiply
+import zzzarchive.composeapp.generated.resources.ui_scale
+import zzzarchive.composeapp.generated.resources.ui_scale_warning
+import kotlin.math.round
+
+@Composable
+fun ScaleFontSizeDialog(
```

Comment:
it a good practice to always pass down a modifier down into your components. It can help you reuse this component and testability.

Reply:
Hi @felipecastilhos , your are my first reviewer on GitHub !!!!!! 

Thanks for your feedback! I created a BasicDialog to make it reusable for other dialogs.

Reply:
Glad to help! I was browsing your repo yesterday and noticed a lot of things I could assist with while I was here